pom.xml

```diff
@@ -5,12 +5,32 @@
 
     <groupId>de.muenchen</groupId>
     <artifactId>refarch</artifactId>
+    <name>refarch</name>
+    <description>Collection of different ready to use RefArch components</description>
     <version>1.1.0-SNAPSHOT</version>
     <packaging>pom</packaging>
-    <name>refarch</name>
 
     <modules>
         <module>refarch-gateway</module>
         <module>refarch-integrations</module>
     </modules>
+
+    <scm>
+        <url>https://github.com/it-at-m/refarch.git</url>
+        <connection>scm:git:https://github.com/it-at-m/refarch.git</connection>
+        <developerConnection>scm:git:https://github.com/it-at-m/refarch.git</developerConnection>
+        <tag>HEAD</tag>
+    </scm>
+    <licenses>
+        <license>
+            <name>MIT</name>
+        </license>
+    </licenses>
+    <developers>
+        <developer>
+            <organization>it@M</organization>
+            <email>opensource@muenchen.de</email>
+            <url>https://github.com/it-at-m</url>
+        </developer>
+    </developers>
```

Comment:
This section is now also at the child `pom.xml`. Is this still necessary?

Reply:
Yes as it's not the parent.